src/inferenceresponse.cpp

```diff
@@ -90,4 +93,9 @@ uint32_t InferenceResponse::getOutputCount() const {
 uint32_t InferenceResponse::getParameterCount() const {
     return this->parameters.size();
 }
+
+void InferenceResponse::Clear() {
+    outputs.clear();
+    parameters.clear();
+}  // TODO remove
```

Comment:
Why do we need to remove it?

Reply:
It's existence in API is required only by tests, so I think it could be good to refactor test in that way. The same is true for no-args constructor.

That would involve quite big refactor in tests so I don't want to do this in single MR.